BlueprintUI/Sources/Layout/Inset.swift

```diff
@@ -158,5 +158,34 @@ extension Inset {
             frame.size.height -= top + bottom
             return LayoutAttributes(frame: frame)
         }
+
+        func sizeThatFits(proposal: SizeConstraint, subelement: LayoutSubelement, cache: inout ()) -> CGSize {
+            let insetProposal = proposal.inset(by: edgeInsets)
+            let childSize = subelement.sizeThatFits(insetProposal)
+            return childSize + CGSize(width: left + right, height: top + bottom)
```

Comment:
Do you think there's any tangible performance gain if we avoid the CGSize arithmetic convenience (since it results in two `CGSize`s being initialized and a function call)?